yt/yt/server/node/exec_node/job_controller.cpp

```diff
@@ -1505,6 +1510,8 @@ class TJobController
 
         auto operationId = job->GetOperationId();
 
+        JobProxyLogManager_->OnJobFinished(job->GetId());
```

Comment:
Since we call this method from `UnregisterJob` it is better to rename `OnJobFinished` to `OnJobUnregistered`